src/onegov/org/layout.py

```diff
@@ -2837,6 +2837,10 @@ def breadcrumbs(self):
             Link(_(self.model.title), self.request.link(self.model))
         ]
 
+    def linkify(self, text):
+        linkified = super().linkify(text)
+        return linkified.replace('\\n', '<br>') if linkified else linkified
```

Comment:
Is the double backslash necessary here? I don't see why it would be, I feel like this actually should produce the wrong result (i.e. replacing the literal string `\n` instead of the newline character).

Reply:
Yes, that's correct. If you look the ticket, we wanted to create a symbol to add newlines. It's the literal string we want.

Reply:
I see